services/analogbutton.md

```diff
@@ -0,0 +1,38 @@
+# Analog Button
+
+    identifier: 0x1865adc9
+    extends: _sensor
+
+A simple pressure sensitive push-button.
+
+## Registers
+
+    ro pressure: u0.16 / @ reading
+
+Indicates the current pressure (``force``) on the button.
+
+    rw low_threshold: u0.16 / @ low_threshold
+    
+Indicates the lower threshold and upper threshold for ``up`` event.
```

Comment:
I'm wondering if we shouldn't use `active` / `inactive` for event names; now we have low threshold for up, and high for down, which seems... upside down. If it's high threshold for active it would make more sense.